Btp operator creds in docs (#1774)
kyma-project/control-plane: docs/kyma-environment-broker/08-01-provisioning-kyma-environment.md

   ```

4. Check the operation status as described [here](08-03-operation-status.md).

## BTP Operator

> If you need a BTP Operator component installed, please obtain [BTP Operator access credentials](https://github.com/SAP/sap-btp-service-operator/blob/v0.2.5/README.md#setup), then provide it in the provisioning request, for example:

Reviewer: ```suggestion
If you need a BTP Operator component installed, obtain [BTP Operator access credentials](https://github.com/SAP/sap-btp-service-operator/blob/v0.2.5/README.md#setup) and provide them in the provisioning request. See the following example:
```
